src/tools/cleaner.js

```diff
@@ -30,27 +30,30 @@ const recurseCleanFuncs = (obj, path) => {
 };
 
 // Recurse a nested object replace all instances of keys->vals in the bank.
-const recurseReplaceBank = (obj, bank) => {
-  bank = bank || {};
+const recurseReplaceBank = (obj, bank = {}) => {
   const replacer = out => {
     if (typeof out !== 'string') {
       return out;
     }
     Object.keys(bank).forEach(key => {
+      // Escape characters (ex. {{foo}} => \\{\\{foo\\}\\} )
       const s = String(key).replace(/[-[\]/{}()\\*+?.^$|]/g, '\\$&');
       const re = new RegExp(s, 'g');
       out = out.replace(re, bank[key]);
     });
+
+    if (obj.omitEmptyParams) {
```

Comment:
Was wondering if we want to prevent `body` fields from being pruned? Hypothetically, APIs _should_ return the same result if a field is empty vs. omitted?